types/stripe-js/elements/card-number.d.ts

```diff
@@ -135,6 +136,44 @@ export interface StripeCardNumberElementOptions {
    * Appearance of the brand icon in the Element.
    */
   iconStyle?: 'default' | 'solid';
+
+  /**
+   * Hides and disables the Link Button in the Card Element.
+   * Default is false.
+   */
+  disableLink?: boolean;
+
+  /**
+   * The preferred card network order for the Card Brand Choice dropdown. Specifying a value for this
+   * parameter will result in the most preferred network being selected by default in the dropdown.
+   * Defaults to the automatic network ordering if no value is specified.
+   */
+  preferredNetwork?: Array<CardNetworkBrand>;
+}
+
+export interface StripeCardNumberElementUpdateOptions {
+  classes?: StripeElementClasses;
+
+  style?: StripeElementStyle;
+
+  placeholder?: string;
+
+  /**
+   * Applies a disabled state to the Element such that user input is not accepted.
+   * Default is false.
```

Comment:
```suggestion
   * Default is `false`.
```
to be consistent with other comments